src/templates/components/trip-card.njk

```diff
@@ -0,0 +1,15 @@
+<a href="/{% if trip.leader === 1 %}leader/{% endif %}trip/{{ trip.id }}" class=trip-card>
```

Comment:
I think this is fine but you will notice that the app in general is a little wary of over-componetizing.  A little copying > a little indirection, sometimes. This is especially true for templates, which usually don't have variable scoping.